content/en/docs/quickstart.md

```diff
@@ -43,86 +43,88 @@ notation list --plain-http $IMAGE
 
 Confirm there are no signatures.
 
-## Generate a certificate
+## Generate a test key and self-signed certificate
 
-Use `notation cert generate-test` to generate a self-signed test certificate for signing artifacts. The following generates a self-signed X.509 certificate under the `~/config/notation/` directory.
+Use `notation cert generate-test` to generate a test RSA key for signing artifacts, and a self-signed test certificate for verifying artifacts.
 
 **IMPORTANT**: Self-signed certificates should be used for development purposes only and should not be used in production environments.
 
+The following command generates a test key and a self-signed X.509 certificate under the `$HOME/.config/notation/localkeys` directory. The test key is set as a default signing key using `--default` flag. The self-signed certificate is added to a named trust store `wabbit-networks.io` of type `ca`.
+
 ```console
 notation cert generate-test --default "wabbit-networks.io"
 ```
 
-The output of the above command shows the location of the public and private key generated. For example:
+Use `notation key list` to confirm the signing key. Key name with a `*` prefix is the default key.
 
-```output
-$ notation cert generate-test --default "wabbit-networks.io"
-generating RSA Key with 2048 bits
-generated certificates expiring on 2022-10-01T20:18:37Z
-wrote key: <EXAMPLE_PATH>/notation/localkeys/wabbit-networks.io.key
-wrote certificate: <EXAMPLE_PATH>/notation/localkeys/wabbit-networks.io.crt
-wabbit-networks.io: added to the key list
-wabbit-networks.io: marked as default
+```console
+notation key list
+```
+
+Use `notation certificate list` to confirm the certificate is stored in the trust store.
+
+```console
+notation cert list
 ```
 
-## Sign the image
+## Sign the container image
 
-Use `notation sign` to sign the container image.
+Use `notation sign` to sign the container image with COSE signature format.
 
 ```console
-notation sign --plain-http $IMAGE
+notation sign --plain-http --signature-format cose $IMAGE
```

Comment:
this command fails for me 

```
notation sign --plain-http --signature-format cose $IMAGE
Error: unknown flag: --signature-format
2022/11/01 15:19:08 unknown flag: --signature-format
```

Reply:
> this command fails for me
> 
> ```
> notation sign --plain-http --signature-format cose $IMAGE
> Error: unknown flag: --signature-format
> 2022/11/01 15:19:08 unknown flag: --signature-format
> ```

Sorry. This is for an upcoming notation release beta.1. The version is not ready yet. It should be ready by Nov 2 PDT.